src/Sylius/Component/User/Context/CustomerContextInterface.php

```diff
@@ -19,7 +19,7 @@
 interface CustomerContextInterface
 {
     /**
-     * @return CustomerInterface|null
+     * @return null|CustomerInterface
```

Comment:
This whole interface makes no sense at all, same as discussing that phpdoc order is correct or no, even if code will first return null not the object. EOT.